app-backend/api/src/main/scala/organization/Routes.scala

```diff
@@ -53,19 +56,18 @@ trait OrganizationRoutes extends Authentication with PaginationDirectives with U
 
   def getOrganization(orgId: UUID): Route = authenticate { user =>
     rejectEmptyResponse {
-      complete {
-        Organizations.getOrganization(orgId)
+      if (user.isInRootOrganization || user.organizationId == orgId) {
```

Comment:
Fixed in https://github.com/azavea/raster-foundry/pull/1383/commits/6bbb3b8f4ceba9beb2dd7e4dc6107659fbfde9f0